test/Amazon.Extensions.CognitoAuthentication.IntegrationTests/CognitoAWSCredentialsTests.cs

```diff
@@ -100,15 +100,15 @@ await identityClient.CreateIdentityPoolAsync(new CreateIdentityPoolRequest()
             AttachRolePolicyResponse attachRolePolicyResponse = managementClient.AttachRolePolicyAsync(attachRequest).Result;
 
             //Set the role for the identity pool
-            await identityClient.SetIdentityPoolRolesAsync(new SetIdentityPoolRolesRequest()
```

Comment:
This test is async, so it should be appropriate to use `await identityClient.SetIdentityPoolRolesAsync`

Reply:
Fixed with new commit. Please review.